src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -78,6 +98,32 @@ def _assess_clusters(self, all_clusters):
                 for value in cluster.spark_conf.values():
                     if "dbfs:/mnt" in value or "/dbfs/mnt" in value:
                         failures.append(f"using DBFS mount in configuration: {value}")
+
+                # Checking if Azure cluster config is present in spark config
+                for key, value in cluster.spark_conf.items():
+                    for conf in _AZURE_SP_CLUSTER_CONF:
+                        if re.search(conf, key):
+                            conf_list += [{key: value}]
+                if conf_list:
+                    cluster_dict["clusterName"] = cluster.cluster_name
+                    cluster_dict["clusterSparkConf"] = conf_list
+                    failures.append(json.dumps(cluster_dict))
```

Comment:
Failures have to be human readable single strings per failure. People would be interested in Top10 failures on the entire account.

Reply:
ok sure. Lets discuss. I will comment these out for now. We are meeting tomorrow and will discuss. These are not really failures. These are not really failures. So we have to store these config somewhere.

Reply:
What do we need to discuss? Failure strings have to be short and concise and informative: "Uses azure service principal credentials in cluster config" and that's it.